__tests__/commands/bin.js

```diff
@@ -0,0 +1,24 @@
+/* @flow */
+
+import {run as buildRun} from './_helpers.js';
+import {BufferReporter} from '../../src/reporters/index.js';
+import {run} from '../../src/cli/commands/bin.js';
+
+const path = require('path');
+
+const fixturesLoc = path.join(__dirname, '..', 'fixtures', 'bin');
+const runBin = buildRun.bind(null, BufferReporter, fixturesLoc, (args, flags, config, reporter): Promise<void> => {
+  return run(config, reporter, flags, args);
+});
+
+test('run bin with no arguments should return the folder where are stored the binaries', (): Promise<void> => {
```

Comment:
`running bin without arguments should return the folder where the binaries are stored`